utils/readsb/Makefile

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2020 Stijn Tintel <stijn@linux-ipv6.be>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=readsb
+PKG_VERSION:=3.8.3
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/Mictronics/readsb/archive/v$(PKG_VERSION)
+PKG_HASH:=bf59b8ecd0ff66945b210c55a5b824aa63ff8cbb1704249528d30a4902e716b2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_BUILD_DEPENDS:=!USE_GLIBC:argp-standalone
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/readsb/default
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libncurses
+  TITLE:=Mode-S/ADSB/TIS decoder for various devices
+  URL:=https://github.com/Mictronic/readsb
+  MAINTAINER:=Stijn Tintel <stijn@linux-ipv6.be>
+endef
+
+define Package/readsb
+  $(call Package/readsb/default)
+  TITLE+= (readsb)
+  DEPENDS+=+librtlsdr
+endef
+
+define Package/viewadsb
+  $(call Package/readsb/default)
+  TITLE+= (viewadsb)
+endef
+
+MAKE_FLAGS += \
+        BLADERF=no \
+        CFLAGS="$(TARGET_CFLAGS)" \
+	RTLSDR=yes \
+        UNAME="Linux"
+
+TARGET_CFLAGS += -ffunction-sections -fdata-sections -flto
+TARGET_LDFLAGS += -Wl,--gc-sections,--as-needed -largp
```

Comment:
-largp probably needs to be conditional. See https://github.com/openwrt/packages/blob/master/libs/libv4l/Makefile#L81

Reply:
Good catch. Fixed.